IdentityCore/src/throttling/cache/MSIDThrottlingCacheRecord.h

```diff
@@ -28,7 +28,7 @@ NS_ASSUME_NONNULL_BEGIN
 
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->